src/hash/mod.rs

```diff
@@ -1,3 +1,5 @@
+#![allow(dead_code)]
+
 mod hash;
```

Comment:
no longer using hashing for the Lru, so hash is currently unused, but I suspect it will find a use later on